README

```diff
@@ -0,0 +1,84 @@
+####################################
```

Comment:
What is the point of this readme?
This is the contributing section of the text....

Reply:
Let's move the content out of the README to an rst page. The README will have a link to the page.